nano/node/confirmation_height_processor.cpp

```diff
@@ -0,0 +1,378 @@
+#include <nano/node/confirmation_height_processor.hpp>
+
+#include <boost/optional.hpp>
+#include <cassert>
+#include <nano/lib/logger_mt.hpp>
+#include <nano/lib/numbers.hpp>
+#include <nano/lib/utility.hpp>
+#include <nano/node/active_transactions.hpp>
+#include <nano/node/stats.hpp>
+#include <nano/secure/blockstore.hpp>
+#include <nano/secure/common.hpp>
+#include <numeric>
+
+nano::confirmation_height_processor::confirmation_height_processor (nano::pending_confirmation_height & pending_confirmation_height, nano::block_store & store, nano::stat & stats, nano::active_transactions & active, nano::block_hash const & epoch_link, nano::logger_mt & logger) :
```

Comment:
May be better name references as store_a, stats_a etc. as it is in other parts of code?

Reply:
Sure, makes sense to be consistent. Changes all function parameters in that file.